code/modules/mob/living/carbon/life.dm

```diff
@@ -43,7 +43,10 @@
 				hud_used.healths.icon_state = "health5"
 			else
 				hud_used.healths.icon_state = "health6"
-
+				
+/mob/living/carbon/human/proc/oncritdrag() //critdragging function, used in signals
```

Comment:
it's a signal handler, so type SIGNAL_HANDLER in the first line of the proc
Autodoc, so do this
///Description
/mob/..../proc/bla

rather than /mob/..../proc/bla //Description

Reply:
And the description is a bit ?? used in signal, which signal? what does the proc do?